must-gather/collection-scripts/gather

```diff
@@ -39,14 +39,16 @@ commands+=("describe nodes")
 commands+=("describe pods -n openshift-storage")
 commands+=("get pvc --all-namespaces")
 commands+=("get pv")
-commands+=("get sc")
 commands+=("get infrastructures.config -oyaml")
 commands+=("get subscription -n openshift-storage")
 commands+=("get csv -n openshift-storage")
+commands+=("get sc")
+commands+=("get installplan -n openshift-storage")
+commands+=("get events -n openshift-storage")
 
 # Run the Collection of Resources to list
 for command in "${commands[@]}"; do
-     echo "collecting oc command ${command}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
+     echo "collecting command oc ${command}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
```

Comment:
Don't include this change. It's not particularly relevant nor does it really add any clarity.

Reply:
ok